packages/web-runtime/src/store/spaces.ts

```diff
@@ -88,8 +88,10 @@ const mutations = {
     state.spaceMembers = members
   },
   UPSERT_SPACE_MEMBERS(state, member) {
+    // group shares don't have the name prop... distinguish by shareType
```

Comment:
Distinction by `shareType` would be ideal, but I don't know if we need the server to adjust as well. Because the server currently handles space user- and group-shares via the same `shareType`: `7`.